include/logging.h

```diff
@@ -86,5 +89,14 @@ void GFX_ShowMsg(char const* format,...) GCC_ATTRIBUTE(__format__(__printf__, 1,
 
 #endif //C_DEBUG
 
+#ifdef DEBUG
+#define DEBUG_LOG_MSG(...)                                                     \
```

Comment:
Very handy! will swap this into `cdrom_image.cpp` after we merge it.

Reply:
Eh, I needed to make definition of this macro more complicated… but, on the plus side, it does not need two `fprintf` calls any more! :)